src/app/pages/maps/search-map/location-service.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
```

Comment:
Let's switch to new HTTP client from @angular/common/http (this also requires some code re-do)